thrift/src/main/thrift/atoms/storyquestions.thrift

```diff
@@ -0,0 +1,27 @@
+namespace * contentatom.storyquestions
+namespace java com.gu.contentatom.thrift.atom.storyquestions
+#@namespace scala com.gu.contentatom.thrift.atom.storyquestions
+
+enum RelatedStoryLinkType {
+  TAG = 0,
+  STORY = 1,
+}
+
+struct Question {
+  1: required string questionId
+  2: required string questionText
+}
+
+struct QuestionSet {
+  1: required string questionSetId
+  2: required string questionSetTitle
+  3: required list<Question> questions
+}
+
+struct StoryQuestionsAtom {
+  1: required string relatedStoryId
+  2: required RelatedStoryLinkType relatedStoryLinkType 
```

Comment:
I'm a bit surprised that a question isn't an atom but instead we have multiple questions in a single atom. Is it the case that you would always want all the questions asked to be bundled up together?